kubernetes/samples/scripts/create-weblogic-domain/model-in-image/create_domain.sh

```diff
@@ -0,0 +1,63 @@
+#!/bin/bash
```

Comment:
In other samples (on-pv and in-image samples), the script that generates the domain resource is create-domain.sh. The doc area that ais general to all domain source types refers to create-domain.sh (see  > User Guide > Manage WebLogic domains > Domain resource). But for MII samples, the script file is named as create_domain.sh. Not sure if it is critical to make it consistent with other samples.

Reply:
I'm in favor of leaving it alone or doing something else:
- Unlike the swiss-army-knife domain-create.sh, this script _only_ creates the domain resource file - it doesn't deploy it or implicitly build an image...
- To reflect this, a better name might be 'stage-domain-resource-file' (similarly 'build_model' would probably be better named 'stage-model-files-for-image.sh')

@rjeberhard What do you think?  Maybe leave things as is and clean up in a later release when we break this sample into even smaller steps/parts?

Reply:
Do we have a style disagreement between underscore and dash?  Otherwise, I'm fine with leaving the content of these samples alone.

Reply:
The > User Guide > Manage WebLogic domains > Domain resource doc has been updated not to mention any script file names. I have not checked who updated it, but the latest looks fine.